src/db/db_place.h

```diff
@@ -47,9 +47,9 @@ namespace db{
             void clear(){
                 _placement.clear();
             }
-            int x(Cell *cell) const {
-                return _placement.at(cell).x();
-                //return _placement[cell].x();
+            int x(Cell *cell) {
+                // return _placement.at(cell).x();
+                return _placement[cell].x();
```

Comment:
I think using the `at` function is safer in case there is no index `cell` in `_placement`.